traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -220,7 +220,7 @@ var TableServersController = function(tableName, servers, filter, $scope, $state
 		},
 		{
 			headerName: "Router Port Name",
-			field: "routerPortName",
+			field: "routerPort",
```

Comment:
This field won't work anymore, since there's no field on the input data named `routerPort` (or `routerPortName`). I think we may just have to remove it.

Reply:
I'm a UI rookie, but isn't the field `interfaceName` working in the same way? It comes from the `interface` struct inside the `server` struct, as do `routerHostName` and `routerPortName`.

Reply:
Yeah, I guess we don't actually need to remove it. It's fair enough to represent the router that's used for the IP addresses that we're adding to it.

But I still think the name is wrong. The property name that gets added from `toLegacyIPInfo` looks like `routerPortName`, so that'd need to be the name of the field to display it properly.

Reply:
yeah I think I changed it to `routerPortName` in my latest commit